packages/client/lib/sentinel/utils.ts

```diff
@@ -0,0 +1,25 @@
+import { RedisNode } from './types';
+
+/* TODO: should use map interface, would need a transform reply probably? as resp2 is list form, which this depends on */
+export function parseNode(data: any): RedisNode | undefined{
+  const flags: string = data[9];
+  if (flags.includes("s_down")) {
+    return undefined;
+  }
+
+  return { "host": data[3], "port": Number(data[5]) };
+}
+
+export function createNodeList(data: any) {
+  var nodeList: Array<RedisNode> = [];
+
+  for (const nodeData of data) {
+    const node = parseNode(nodeData)
+    if (node === undefined) {
```

Comment:
undefined is like false in node - can we do an if/else here?

Reply:
what do you mean?  in this case, I have parseNode return undefined for a node that is down, so can ignore it.